src/GraphQL/Execution/ExecutionHelper.cs

```diff
@@ -105,12 +106,12 @@ public static object GetVariableValue(Document document, ISchema schema, Variabl
 
             try
             {
-                AssertValidValue(schema, type, input, variable.Name);
+                AssertValidVariableValue(schema, type, input, variable.Name);
             }
             catch (InvalidValueException error)
             {
                 error.AddLocation(variable, document);
-                throw;
+                throw error;
```

Comment:
Why is this done?

Reply:
I don't know.  I just verified that the location is added regardless of `throw` or `throw error`.  I'll fix it.

Reply:
```suggestion
                throw;
```

Reply:
`throw error` loses the stack